service/src/auth/auth.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from "@nestjs/jwt";
+import { UsersService } from '../users/users.service';
+
+
+@Injectable()
+export class AuthService {
+    constructor(
+        private usersService:UsersService, 
+        private jstService:JwtService){}
+
+    async signIn(email:string, pass: string): Promise<any>{
+        const user = await this.usersService.findOne(undefined, email)
+        if (user?.password !== pass){
+            throw new UnauthorizedException();
+        }
+        const token_data = {
```

Comment:
Unnecessary usage of snake case, might be a good idea to modify it in camel case to match the rest of the project

Reply:
I accept the comment.